py2many/exceptions.py

```diff
@@ -0,0 +1,8 @@
+import ast
+
+
+class AstNotImplementedError(NotImplementedError):
```

Comment:
Perhaps make this a base class for capturing the lineno and then have `TypeError` and `AstNotImplementedError` as subclasses.